osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -31,14 +34,24 @@ public partial class PauseOverlay : GameplayMenuOverlay
                 OnResume?.Invoke();
         };
 
+        private IBindable<bool>? windowActive;
+
+        private float targetVolume => windowActive?.Value != false && State.Value == Visibility.Visible ? 1.0f : 0;
+
         [BackgroundDependencyLoader]
-        private void load()
+        private void load(GameHost? host)
         {
             AddInternal(pauseLoop = new SkinnableSound(new SampleInfo("Gameplay/pause-loop"))
             {
                 Looping = true,
                 Volume = { Value = 0 }
             });
+
+            if (host != null)
+            {
+                windowActive = host.IsActive.GetBoundCopy();
+                windowActive.BindValueChanged(_ => Schedule(() => pauseLoop.VolumeTo(targetVolume, 1000, Easing.Out)));
```

Comment:
`BindValueChanged()` callback in BDL rather than in `LoadComplete()`? Also what's with the schedule, is it a workaround for this being in BDL?

Reply:
The schedule is very unfortunate. It seems the window active flag is not always on the update thread...

![JetBrains Rider-EAP 2025-03-19 at 10 55 36](https://github.com/user-attachments/assets/a44fd516-5aa8-49be-9f03-29bd2be0353f)

Reply:
I figured that might have been the case as well, sure.